ooo-request/README.md

```diff
@@ -0,0 +1,87 @@
+### OOO Status request Firestore collection model
+
+```json
+{
+  "id": "String",
+  "userId": "String",
+  "state": "PENDING",
+  "from": "Timestamp",
+  "until": "Timestamp",
+  "message": "String",
+  "createdAt": "Timestamp",
+  "updatedAt": "Timestamp",
+  // after approval
+  "state": "APPROVED",
+  "approvedAt": "Timestamp",
+  "approvedBy": "String",
+  // after rejection
+  "state": "REJECTED",
+  "rejectedAt": "Timestamp",
+  "rejectedBy": "String",
+  "rejectedReason": "String"
```

Comment:
- I think we should keep same field for `approvedBy` and `rejectedBy`. eg: `lastModifiedBy`
- Also think `rejectedReason` can be just `reason`.

Reply:
`processedBy` ?